Resources/skeleton/web/config.php

```diff
@@ -12,7 +12,7 @@
     exit('This script is only accessible from localhost.');
 }
 
-require_once dirname(__FILE__).'/../app/SymfonyRequirements.php';
+require_once __DIR__.'/../app/SymfonyRequirements.php';
```

Comment:
IITC, this was done on purpose, because one of the checks is that PHP 5.3+ is used.